README.md

```diff
@@ -35,20 +35,21 @@ It offers a variety of features:
 
 ## Apache RocketMQ Community
 * [RocketMQ Streams](https://github.com/apache/rocketmq-streams): A lightweight stream computing engine based on RocketMQ.
-* [RocketMQ Flink](https://github.com/apache/rocketmq-flink): The RocketMQ connector of Flink that supports source and sink connector in data stream and Table. 
-* [RocketMQ Client CPP](https://github.com/apache/rocketmq-client-cpp)
-* [RocketMQ Client Go](https://github.com/apache/rocketmq-client-go)
+* [RocketMQ Flink](https://github.com/apache/rocketmq-flink): The RocketMQ connector of Flink that supports source and sink connector in data stream and Table.
+* [RocketMQ Client CPP](https://github.com/apache/rocketmq-client-cpp): RocketMQ CPP Client.
+* [RocketMQ Client Go](https://github.com/apache/rocketmq-client-go): RocketMQ Go Client.
 * [RocketMQ Client Python](https://github.com/apache/rocketmq-client-python)
 * [RocketMQ Client Nodejs](https://github.com/apache/rocketmq-client-nodejs)
 * [RocketMQ Spring](https://github.com/apache/rocketmq-spring)
 * [RocketMQ Exporter](https://github.com/apache/rocketmq-exporter)
-* [RocketMQ Operator](https://github.com/apache/rocketmq-operator)
-* [RocketMQ Docker](https://github.com/apache/rocketmq-docker)
-* [RocketMQ Dashboard](https://github.com/apache/rocketmq-dashboard): Operation and maintenance console of RocketMQ
-* [RocketMQ Connect](https://github.com/apache/rocketmq-connect): A tool for scalably and reliably streaming data between Apache RocketMQ and other systems
-* [RocketMQ MQTT](https://github.com/apache/rocketmq-mqtt)
-* [RocketMQ Incubating Community Projects](https://github.com/apache/rocketmq-externals)
+* [RocketMQ Operator](https://github.com/apache/rocketmq-operator): Provides a way to run an RocketMQ cluster on Kubernetes.
```

Comment:
1. It seems `Providing` should be used instead of `Provides`.
2. `an Apache RocketMQ cluster` is more appropriate.